dask/dataframe/tests/test_rolling.py

```diff
@@ -548,12 +548,25 @@ def test_groupby_rolling():
     expected = df.groupby("group1").rolling("15D").sum()
     actual = ddf.groupby("group1").rolling("15D").sum()
 
-    assert_eq(expected, actual, check_divisions=False)
+    check_dtype = not pyarrow_strings_enabled()
```

Comment:
Why is this needed?

Reply:
It shouldn't be. I added this to other `MultiIndex` cases to follow-up on here: https://github.com/dask/dask/issues/10025.